packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -145,14 +184,14 @@ const deployContract = async (
     constructorCalldata,
   });
 
-  console.log("Contract Deployed at ", contractAddress);
+  console.log(green("Contract Deployed at "), contractAddress);
 
-  let finalContractName = exportContractName || contractName;
+  let finalContractName = contract || contractName;
```

Comment:
the `contractName` variable comes first, if its set, it shoulld take this one as it is the exportName indicated by the user.